vrchat.jordo.easyquestswitch/Runtime/Resources/EQS_Localizations/es.json

```diff
@@ -21,6 +21,7 @@
 	"SettingsHierarchy": "Ajustes de jerarquía",
 	"SettingsHierarchyIconShow": "Mostrar icono en la jerarquía",
 	"SettingsHierarchyIconOffset": "Ajuste de posición del icono",
+	"SettingsPromptForPlatformChange": "Show prompt to apply EQS changes if a newly opened scene still uses setting from the opposite platform (Can cause issues with Bakery).",
```

Comment:
@RadAurum If you are available, would you have a translation to suggest for this?